.gitignore

```diff
@@ -2,6 +2,9 @@
 #-----------------------------------
 build/
 darkreader.js
+tests/inject/**/*.tests.js
+tests/inject/customize.js
+tests/inject/polyfills.js
```

Comment:
I don't think they should be gitignored, they are not generated and are required for the tests.

Reply:
I don't seem to be having that behavior, or is this caused by the upgraded package?

```
-> % node --version
v16.13.2
-> % npm --version
8.4.0
```

Reply:
To be clear, you're referring to having JS files next to the TS? Where are yours placed when they are generated? Do they just remain in memory, then passed to Karma? That's what it seemed to be doing before.

Reply:
I can't think how the node/npm versions would be affecting it, but who knows? 

```
% node --version; npm --version
v14.18.1
8.3.0
```

Reply:
> I don't seem to be having that behavior,

Wait, are you talking about the inability to update when autowatch is active?

Reply:
> Wait, are you talking about the inability to update when autowatch is active?

No, the generated files aren't being stored on disk... autowatch still seems to be working with this PR. I just don't have the behavior that the generated files are being stored on the disk.